internal/pkg/dynamic-config/dynamic_context.go

```diff
@@ -183,7 +183,7 @@ func (d *DynamicContext) SchemaRegistryCluster(cmd *cobra.Command) (*v1.SchemaRe
 			if len(srClusters) != 0 {
 				cluster = makeSRCluster(&srClusters[0])
 			} else {
-				cluster = nil
+				return nil, errors.NewSRNotEnabledError()
```

Comment:
There's a comment at the start of this function saying that it returns an empty SchemaRegistryCluster if none is set. I think we should probably change that to say that this will return an `SRNotEnabledError` if none is set.

I skimmed over the places in the code that call this function and it looks like all of them expect a non-nil SR cluster, and some of them even looked for `SRNotEnabledError`s; can you double check before merge?

Reply:
Sounds good, changed the comment.

Double checked the usages, whenever the function returns a nil SR cluster, it's always coupled with an error so we should be fine there.